### DISCUSSION TITLE
Previewing posts from the Wordpress admin

### DISCUSSION THREAD

**Discussion by kylewindfall** [corcel/corcel#564] [Q&A]
Does anyone have a good solution for previewing posts from the Wordpress admin?

**Comment by bdahlinger** [comment]
You can modify the existing theme to kind of match your main site look, not ideal. 

You can use [roots.io sage](https://roots.io/sage/) maybe alongside [bedrock ](https://roots.io/bedrock/) and have more custom control over the custom template, not ideal. 

I've done so many of these on Laravel now but haven't invested the time into writing an WP override function to open the main site that uses Corcel and handle previewing. Seems doable I think. I just end up showing them that dang default theme preview you get built into WP, and don't get me started on Gutenberg blocks rendering in Laravel, what a pain!
